services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -407,4 +378,37 @@ private void setUpIndividualRequest(final IIndividualRequest request) {
         request.setBulkSubmission(bulkSubmission);
     }
 
+    @Test
+    public void setIndividualRequestDaoTest() throws Exception {
+
+        messageTaskService.setIndividualRequestDao(mockIndividualRequestDao);
+
+        Object result = this.getAccessibleField(RetryMessageAlreadySentService.class, "individualRequestDao",
+                                                IIndividualRequestDao.class, messageTaskService);
+
+        assertEquals(mockIndividualRequestDao, result, INDIVIDUAL_REQUEST_DAO_SHOULD_BE_SET_CORRECTLY);
+    }
+
+    @Test
+    public void setMessagingUtilityTest() throws Exception {
+
+        messageTaskService.setMessagingUtility(mockMessagingUtility);
+
+        Object result = this.getAccessibleField(RetryMessageAlreadySentService.class, "messagingUtility",
+                                                IMessagingUtility.class, messageTaskService);
+
+        assertEquals(mockMessagingUtility, result, INDIVIDUAL_REQUEST_DAO_SHOULD_BE_SET_CORRECTLY);
```

Comment:
The constant used for the message references an IndividualRequestDao object whereas this test is testing the assignment of a MessagingUtility object.  Change message to one that references the right object type.

The same applies to the following setGlobalParametersCacheTest() test.  You'll probably find you don't need the constant after changing the messages as it'll be only used in the setIndividualRequestDaoTest() test.

Reply:
done